buildSrc/src/main/resources/pom.xml

```diff
@@ -12,6 +12,11 @@
         </repository>
     </repositories>
     <dependencies>
+        <dependency>
+            <groupId>io.micronaut.aws</groupId>
+            <artifactId>micronaut-aws-secretsmanager</artifactId>
+            <version>2.8.0</version>
+        </dependency>
```

Comment:
Please open an issue to remove this and the version in https://github.com/micronaut-projects/micronaut-guides/pull/239/files#diff-9e407ac63e06eef118abe440abb54fc5616fa8dca247ded595151e98060ac742R27 once 3.8.0 is included in the BOM.